tests/acceptance/course-page/publish-to-github-test.js

```diff
@@ -16,7 +16,7 @@ module('Acceptance | course-page | publish-to-github-test', function (hooks) {
 
   test('can initiate GitHub integration setup', async function (assert) {
     testScenario(this.server);
-    signInAsStaff(this.owner, this.server);
+    signInAsSubscriber(this.owner, this.server); // Temporarily using subscriber to verify dark mode
```

Comment:
We can remove the changes in this file now, we only needed them to testing locally